ntoskrnl/cc/cacheman.c

```diff
@@ -137,17 +138,20 @@ CcGetFlushedValidData (
 }
 
 /*
- * @unimplemented
+ * @implemented
```

Comment:
That's not really what I would call an "implementation" 🤣

Reply:
So what's this then? :confused: I should keep `unimplemented` there?

Reply:
Two options: either just remove this comment block, or (better) write proper SAL annotation for it, so it can be properly Doxyged'ed.

Reply:
Is there any example how to write SAL annotation properly? :thinking:

Reply:
Done. :ok_hand: